client.go

```diff
@@ -644,6 +644,21 @@ func (s *Client) LoopInQuote(ctx context.Context,
 	}, nil
 }
 
+// GetLiquidityConfig gets the currently set parameters for our liquidity
+// manager.
+func (s *Client) GetLiquidityConfig(ctx context.Context) (*liquidity.Parameters,
+	error) {
+
+	return s.manager.GetParameters(ctx)
```

Comment:
Is this pass through really needed or can `swapClientServer` also invoke manager directly?